packages/rocketchat-slashcommands-msg/server.coffee

```diff
@@ -0,0 +1,36 @@
+###
+# Msg is a named function that will replace /msg commands
+###
+
+class Msg
+	constructor: (command, params, item) ->
+		if command isnt 'msg' or not Match.test params, String
+			return
+
+		trimmedParams = params.trim()
+
+		usernameOrig = trimmedParams.slice(0, trimmedParams.indexOf(' '))
+		message = trimmedParams.slice(trimmedParams.indexOf(' ') + 1)
+
+		username = usernameOrig.replace('@', '')
+
+		if username is ''
+			return
+
```

Comment:
You should check if the message is not empty and if the number of parameters is > 1. Currently, if I type `/msg marcelo`, it will send the message 'marcelo' to user named 'marcel'.